op-chain-ops/foundry/artifact.go

```diff
@@ -153,3 +155,26 @@ func ReadArtifact(path string) (*Artifact, error) {
 	}
 	return &artifact, nil
 }
+
+// SearchRemappings applies the configured remappings to a given source path.
+// It assumes that each remapping is of the form "alias/=actualPath".
+func (a Artifact) SearchRemappings(sourcePath string) string {
```

Comment:
Should this return an error if there's no remapping found, or is the idea that it returns the `sourcePath` if there's no remapping found?

Reply:
Should just return `sourcePath` if no remapping is found. Updated the function comment [here](https://github.com/ethereum-optimism/optimism/pull/14359/commits/510e3598be39e2763f1b5c02c4cc836fad7ecf47) to make that more clear